scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile

```diff
@@ -0,0 +1,117 @@
+
+
+
+#download and extract boost
+RUN powershell.exe -Command \
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    wget https://dl.bintray.com/boostorg/release/1.69.0/source/boost_1_69_0.zip -OutFile c:\TEMP\boost.zip; \
+    mkdir c:\boost; \
+    c:\7zip\7z.exe x c:\TEMP\boost.zip -o"c:\boost"
+
+#Install python 3.5
+RUN powershell.exe -Command \
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    wget https://www.python.org/ftp/python/3.5.4/python-3.5.4-amd64.exe -OutFile c:\temp\python35.exe; \
+    mkdir c:\python\35; \
+    Start-Process c:\temp\python35.exe -ArgumentList '/quiet InstallAllUsers=1 PrependPath=0 TargetDir=c:\\python\\35' -Wait; \
+    c:\python\35\python.exe -m pip install --upgrade pip; \
+    c:\python\35\python.exe -m pip install --upgrade setuptools wheel
+
+#Install python 3.6
+RUN powershell.exe -Command \
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    wget https://www.python.org/ftp/python/3.6.8/python-3.6.8-amd64.exe -OutFile c:\temp\python36.exe; \
+    mkdir c:\python\36; \
+    Start-Process c:\temp\python36.exe -ArgumentList '/quiet InstallAllUsers=1 PrependPath=0 TargetDir=c:\\python\\36' -Wait; \
+    c:\python\36\python.exe -m pip install --upgrade pip; \
+    c:\python\36\python.exe -m pip install --upgrade setuptools wheel
+
+
+#Install python 3.7
+RUN powershell.exe -Command \
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    wget https://www.python.org/ftp/python/3.7.7/python-3.7.7-amd64.exe -OutFile c:\temp\python37.exe; \
+    mkdir c:\python\37; \
+    Start-Process c:\temp\python37.exe -ArgumentList '/quiet InstallAllUsers=1 PrependPath=0 TargetDir=c:\\python\\37' -Wait; \
+    c:\python\37\python.exe -m pip install --upgrade pip; \
+    c:\python\37\python.exe -m pip install --upgrade setuptools wheel
+
+#Install python 3.8
+RUN powershell.exe -Command \
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    wget https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe -OutFile c:\temp\python38.exe; \
+    mkdir c:\python\38; \
+    Start-Process c:\temp\python38.exe -ArgumentList '/quiet InstallAllUsers=1 PrependPath=0 TargetDir=c:\\python\\38' -Wait; \
+    c:\python\38\python.exe -m pip install --upgrade pip; \
+    c:\python\38\python.exe -m pip install --upgrade setuptools wheel
+
+#Downloand blas/lapack
+RUN powershell.exe -Command \
+	[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+	mkdir c:\libs; \
+	wget https://icl.cs.utk.edu/lapack-for-windows/libraries/VisualStudio/3.7.0/Dynamic-MINGW/Win64/libblas.lib -OutFile c:\libs\libblas.lib; \
+	wget https://icl.cs.utk.edu/lapack-for-windows/libraries/VisualStudio/3.7.0/Dynamic-MINGW/Win64/liblapack.lib -OutFile c:\libs\liblapack.lib
+
+#Download eigenlib
```

Comment:
This is contained in the App itself meanwhile, you can remove it here

Reply:
oh, that's nice

Reply:
I meant the Eigen lib, Blas and Lapack are still needed (only for the ExternalSolversApplication)

Reply:
In that case I'm lost, why would I want to remove them?

Reply:
To me it seems like you removed Blas & Lapack in your last commit: https://github.com/KratosMultiphysics/Kratos/pull/6633/commits/f4a6f16dba4d5b572cc0cb5c20b8d0261d40c44b

Reply:
Which was a mistake, since they are still needed by ExternalSolvers

Reply:
@philbucher  Actually if we go ahead with the Eigensolvers app as replacement for the External we could remove those once SuperLu is replaced everywhere. Maybe we can speak it this afternoon, it's still not clear to me if there is still some need for it

Reply:
I suggest we clean this up after [this](https://github.com/KratosMultiphysics/Kratos/pull/6482#issuecomment-608456537) is done